### PR TITLE
Adding installation instructions for Linux and MacOS

### PR DESCRIPTION
**Pull request by yohgenius**
#113  Adding installation instructions for Linux and MacOS for scala language

### CODE REVIEW

File: Scala/README.md [hunk 1]
```diff
@@ -2,11 +2,27 @@
...
-Before you start installing Scala on your machine, you must have Java 1.8 or greater installed on your computer.
+Before you start installing Scala on your machine, you must have Java 1.8 or greater installed on your computer. Type `java -version` to check the Java version.
+
+
 
 ### Installing Scala binaries
 
 * Windows: Install [Scala binaries](http://www.scala-lang.org/download/).
+* Linux: Install  
+&nbsp;&nbsp;&nbsp;&nbsp; - Ubuntu and other Debian-based distro
```
Comment (sbshah97) [inline Scala/README.md]: Remove `&nbsp;` Use `#` if you want to divide the article!

File: Scala/README.md [hunk 2]
```diff
@@ -2,11 +2,27 @@
...
+&nbsp;&nbsp;&nbsp;&nbsp; - Ubuntu and other Debian-based distro  
+                   `echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list`  
+                    `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823`  
+                    `sudo apt-get update`  
+                    `sudo apt-get install sbt`  
+ &nbsp;&nbsp;&nbsp;&nbsp; - Red Hat Enterprise Linux and other RPM-based distro  
+                 `curl https://bintray.com/sbt/rpm/rpm | sudo tee/etc/yum.repos.d/bintray-sbt-rpm.repo`  
+                 `sudo yum install sbt`  
+* MacOS: Install  
+&nbsp;&nbsp;&nbsp;&nbsp;- [Using Homebrew](https://brew.sh/)
```
Comment (sbshah97) [inline Scala/README.md]: Remove `&nbsp;` Use `#` if you want to divide the article!

File: Scala/README.md [hunk 3]
```diff
@@ -2,11 +2,27 @@
...
+                    `$ brew install sbt`  
+&nbsp;&nbsp;&nbsp;&nbsp;- [Using Macports](https://www.macports.org/)
```
Comment (sbshah97) [inline Scala/README.md]: Remove `&nbsp;` Use `#` if you want to divide the article!

File: Scala/README.md [hunk 4]
```diff
@@ -2,11 +2,27 @@
...
 ### Installing Scala binaries
 
 * Windows: Install [Scala binaries](http://www.scala-lang.org/download/).
+* Linux: Install  
+&nbsp;&nbsp;&nbsp;&nbsp; - Ubuntu and other Debian-based distro  
+                   `echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list`  
+                    `sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2DF73499E82A75642AC823`  
+                    `sudo apt-get update`  
+                    `sudo apt-get install sbt`  
+ &nbsp;&nbsp;&nbsp;&nbsp; - Red Hat Enterprise Linux and other RPM-based distro
```
Comment (sbshah97) [inline Scala/README.md]: Remove `&nbsp;` Use `#` if you want to divide the article!

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Please reference the Issue as well! Also specify in the Pull Request description which language this is for?

File: Scala/README.md (hunk 3 above)
Comment (yohgenius) [reply Scala/README.md]: thanks for the instructions... i will fix it again.

**Review by sbshah97 — APPROVED** [review]

**Pull request merged**